.github/workflows/ci.yml

```diff
@@ -294,6 +294,7 @@ jobs:
 
       - name: Upload coverage report
         uses: actions/upload-artifact@v3
+        timeout-minutes: 1
```

Comment:
```suggestion
        timeout-minutes: 1
        continue-on-error: true
```
we need to mark the job as successful even if this step fails: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error